internal/pkg/dl/unmarshal.go

```diff
@@ -0,0 +1,22 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package dl
+
+import (
+	"encoding/json"
+	"fleet/internal/pkg/es"
+	"fleet/internal/pkg/model"
+)
+
+func Unmarshal(hit es.HitT, v interface{}) error {
```

Comment:
Why not put this on the actual type that way `json.Unmarshal` will do it for you?

```
func (hit *es.HitT) Unmarshal(data []byte) error {
   ..unmarshal..
}
```

Reply:
Sure, will update